docs/Typical_workflow_example_with_WRF.rst

```diff
@@ -26,30 +26,28 @@ Searching for build specification for a particular software package can be done
 for example, to get a list of available easyconfig files for WRF::
 
   $ eb -S WRF
-  == temporary log file in case of crash /tmp/easybuild-MdAp7p/easybuild-zEBJMk.log
-  == Searching (case-insensitive) for 'WRF' in /home/example/.local/easybuild/software/EasyBuild/1.15.2/lib/python2.7/site-packages/easybuild_easyconfigs-1.15.2.0-py2.7.egg/easybuild/easyconfigs
-  CFGS1=/home/example/.local/easybuild/software/EasyBuild/1.15.2/lib/python2.7/site-packages/easybuild_easyconfigs-1.15.2.0-py2.7.egg/easybuild/easyconfigs/w/WRF
-   * $CFGS1/WRF-3.3.1-goalf-1.1.0-no-OFED-dmpar.eb
-   * $CFGS1/WRF-3.3.1-goolf-1.4.10-dmpar.eb
-   [...]
-   * $CFGS1/WRF-3.5-goolf-1.4.10-dmpar.eb
-   * $CFGS1/WRF-3.5-ictce-4.1.13-dmpar.eb
-   * $CFGS1/WRF-3.5-ictce-5.3.0-dmpar.eb
-   * $CFGS1/WRF-3.5.1-goolf-1.4.10-dmpar.eb
-   * $CFGS1/WRF-3.5.1-goolf-1.5.14-dmpar.eb
-   [...]
-  == temporary log file /tmp/easybuild-MdAp7p/easybuild-zEBJMk.log has been removed.
-  == temporary directory /tmp/easybuild-MdAp7p has been removed.
-
-Various easyconfig files are found: for different versions of WRF (e.g., v3.5 and v3.5.1),
-for different (versions of) compiler toolchains (e.g., goolf v1.4.10, goolf v1.5.14, ictce), etc.
-
-For the remainder of this example, we will use the available ``WRF-3.5.1-goolf-1.4.10-dmpar.eb``
+  CFGS1=/home/example/.local/easybuild/software/EasyBuild/4.1.1/easybuild/easyconfigs
+   * $CFGS1/w/WPS/WPS-4.0.1_find-wrfdir.patch
+   * $CFGS1/w/WPS/WPS-4.0.2_find-wrfdir.patch
+   [ . . . ]
+   * $CFGS1/w/WRF/WRF-4.0.1-intel-2018b-dmpar.eb
+   * $CFGS1/w/WRF/WRF-4.0.2-foss-2018b-dmpar.eb
+   [ . . . ]
+  
+  Note: 16 matching archived easyconfig(s) found, use --consider-archived-easyconfigs to see them
+
+
+
+Various easyconfig files are found: for different versions of WRF (e.g., v3.8 and v4.1.3),
```

Comment:
@justbennet I would mention versions here that are shown in the output above? So `v4.0.1` and `v4.0.2`?

Reply:
I believe I made this change in the second commit of this PR.